site/content/blog/carvelizing-helm-chart.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "Carvelizing Helm Chart"
+slug: carvelize-helm-chart
+date: 2022-02-16
+author: Rohit Aggarwal
+excerpt: "Use Carvel to author and consume helm chart "
+image: /img/logo.svg
+tags: ['carvel', 'helm', 'gitops']
+---
+# Carvelizing Helm Chart
+
+## Introduction
+
+In this blog, we will see how to author [`bitnami nginx helm chart`](https://github.com/bitnami/charts/tree/master/bitnami/nginx) into Carvel package and then consume the same.
+
+## Why should I choose Carvel
```

Comment:
This heading can also go

Reply:
I have added it to pitch a case for Carvel. This also set the precedence why to Carvel package an helm chart.